apps/src/lib/levelbuilder/lesson-editor/ActivitySectionCardButtons.jsx

```diff
@@ -79,8 +86,22 @@ export default class ActivitySectionCardButtons extends Component {
   };
 
   handleCloseAddTip = tip => {
-    if (!this.state.editingExistingTip) {
-      this.props.addTip(tip);
+    /*
+      If no tip was provided then user exited without saving
+      and we should not update the tips
+    */
```

Comment:
I just realized upon reading this that there is no cancel button. I hope no one presses delete, expecting the cancel button to be there! 😛  it seems a bit non-standard but again I'm ok with it if we're sure that's how we want it.

Reply:
Ya the cancel is the X in the top corner or clicking outside the dialog but I can see how having a cancel button would be good too. @tess323 thoughts on the best design here?

Reply:
Also any thoughts @tess323 about if we should have a confirmation dialog before deleting the tip?

Reply:
@dmcavoy  - I like the confirmation dialog idea - - I think that would negate accidentally deleting when trying to close the window.  I would maybe also darken the color of the 'x' in the right to draw the eye.  (Also this looks great)

Reply:
@davidsbailey @tess323 the x is part of the Dialog component that is used across our site. Is it ok to update it?